android/src/main/java/io/wazo/callkeep/RNCallKeepModule.java

```diff
@@ -479,7 +481,7 @@ private void registerPhoneAccount(Context appContext) {
         String appName = this.getApplicationName(this.getAppContext());
 
         PhoneAccount.Builder builder = new PhoneAccount.Builder(handle, appName)
-                .setCapabilities(PhoneAccount.CAPABILITY_CALL_PROVIDER);
+                .setCapabilities(PhoneAccount.CAPABILITY_CALL_PROVIDER  | PhoneAccount.CAPABILITY_VIDEO_CALLING);
```

Comment:
This makes the phone account disappear on restart. Just investigating this myself and have tracked it down to this line - even changing it to PhoneAccount.CAPABILITY_SUPPORTS_VIDEO_CALLING breaks it.

Reply:
But based on https://android.googlesource.com/platform/cts/+/cc920ce%5E%21/ this should be allowed.

If you enable this,  enable the calling account etc, and then restart the phone your calling account won't be there until you launch the app.

Reply:
Huh... it appears on first look of the settings and then disappears!  So maybe the above is wrong...